core/src/main/java/io/undertow/UndertowOptions.java

```diff
@@ -112,13 +112,18 @@ public class UndertowOptions {
      */
     public static final Option<Integer> MAX_COOKIES = Option.simple(UndertowOptions.class, "MAX_COOKIES", Integer.class);
 
+    /**
+     * Default value of {@link #ALLOW_EQUALS_IN_COOKIE_VALUE} option.
```

Comment:
DEFAULT_ALLOW_ENCODED_SLASH. Please update the comment

Reply:
Updated!